pkg/test/conformance/metadata.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conformance
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path"
+
+	"sigs.k8s.io/yaml"
+)
+
+// MetadataFileName is the canonical name of the metadata file.
+const MetadataFileName = "test.yaml"
+
+// Metadata about a single test
+type Metadata struct {
+	// Name of the test, if specified.
+	Name string
+
+	// Skip the test
+	Skip bool `json:"skip"`
+
+	// Run tests in isolated (non-parallel) mode
+	Isolated bool `json:"isolated"`
```

Comment:
Maybe just call this `Parallel`, unless there is something else implied by `Isolated`?

Reply:
I started by calling this `Parallel` first, but I wanted to file to have good defaults (i.e. enable parallelism by default).

Reply:
I imagine parallel is a good default for native environment, but is it for kube?

BTW if it's just Isolated sounding weird, Exclusive may be another option.

Reply:
It is a good default for both. The match mechanism/format is designed precisely to handle parallel execution mode.

Reply:
renamed isolated => exclusive.